test-utils/loadtester/src/main.rs

```diff
@@ -110,7 +110,7 @@ fn main() {
             Arg::with_name("home")
             .long("home")
             .takes_value(true)
-            .default_value(&default_home)
+            .default_value(default_home.to_str().unwrap())
```

Comment:
There are a lot of `default_home.to_str().unwrap()`, which does not feel optimal. Do we actually need to change default_home to a `PathBuf`?

Reply:
agree, yet @matklad's argument
> it's useful to know which things are paths

still feels relevant and quite good.

Reply:
```suggestion
            .default_value_os(default_home.as_os_str())
```

Reply:
fixed